src/gravel/tests/conftest.py

```diff
@@ -97,21 +94,67 @@ def _get_data_contents(dir: str, fn: str):
 
 
 @pytest.fixture()
-def gstate(mocker: MockerFixture):
-    mocker.patch('gravel.controllers.config.Config')
-    mocker.patch('gravel.controllers.kv.KV')
-    from gravel.controllers.gstate import gstate as _gstate
-    yield _gstate
+@pytest.mark.asyncio
+async def gstate(fs: fake_filesystem.FakeFilesystem, mocker: MockerFixture):
+    mock_ceph_modules(mocker)
+    mock_aetcd_modules(mocker)
+
+    from gravel.controllers.nodes.mgr import NodeMgr
+    from gravel.controllers.resources.inventory import Inventory
+    from gravel.controllers.resources.devices import Devices
+    from gravel.controllers.resources.status import Status
+    from gravel.controllers.resources.storage import Storage
+    from gravel.controllers.services import Services, ServiceModel
+
+    class FakeServices(Services):
+        async def _save(self):
+            pass
 
+        async def _load(self):
+            pass
 
-@pytest.fixture()
-def services(gstate: Generator[Any, None, None], mocker: MockerFixture):
-    class MockStorage(mocker.MagicMock):  # type: ignore
+        def _create_cephfs(self, svc: ServiceModel):
+            pass
+
+        def _create_nfs(self, svc: ServiceModel):
+            pass
+
+        def _is_ready(self):
+            return True
+
+    class FakeStorage(Storage):  # type: ignore
         available = 2000
-    mocker.patch('gravel.controllers.resources.storage', MockStorage)
-    mocker.patch('gravel.controllers.services.Services._save')
-    mocker.patch('gravel.controllers.services.Services._load')
-    mocker.patch('gravel.controllers.services.Services._create_service')
-    from gravel.controllers.services import Services
-    services = Services()
-    yield services
+        total = 2000
+
+    gstate: GlobalState = GlobalState()
+
+    # init node mgr
+    nodemgr: NodeMgr = NodeMgr(gstate)
+
+    probe_interval = gstate.config.options.devices.probe_interval
```

Comment:
same as before: we have options for each controller's probe interval, we should probably be using those.